src/chapters/errors-and-debugging/error-types.rst

```diff
@@ -0,0 +1,93 @@
+Error Types
+===========
+
+.. index::
+   single: error; syntax
+   single: error; type
+   single: error; runtime
+   single: object; built-in
+
+An **error type** is the classification that JavaScript uses internally to group errors based on their cause. In future lessons, we will come to understand that an error type is actually something called a **built-in object**. For now, however, understanding the different types of errors will help us become faster at debugging.
+
+Each error that JavaScript reports has an error type, and the type is included in the error message. For example, one of the messages we saw previously reported the error type as ``SyntaxError``.
```

Comment:
Word removal: "For example, an earlier message reported the error type as ``SyntaxError``."

Could we make 'an earlier message' a link to the relevant page and section (6.3.1, I think)?

Reply:
Good suggestion. I added a link.